CMakePresets.json

```diff
@@ -270,7 +271,17 @@
       "cacheVariables": {
         "CCCL_ENABLE_C": true,
         "CCCL_C_Parallel_ENABLE_TESTING": true,
-        "CCCL_C_Parallel_ENABLE_HEADER_TESTING": true
+        "CCCL_C_Parallel_ENABLE_HEADER_TESTING": true,
+        "CCCL_C_EXPERIMENTAL_STF_ENABLE_TESTING": true
+      }
+    },
+    {
+      "name": "cccl-c-stf",
+      "displayName": "CCCL C CUDASTF Library",
+      "inherits": "base",
+      "cacheVariables": {
+        "CCCL_ENABLE_C": true,
+        "CCCL_C_EXPERIMENTAL_STF_ENABLE_TESTING": true
```

Comment:
Similar to above, disable the parallel stuff so we don't redundantly test things.